robot.py

```diff
@@ -157,6 +157,8 @@ def teleopPeriodic(self):
         '''Called on each iteration of the control loop'''
         self.putData()
 
+        self.compressor.setClosedLoopControl(self.chassis.compressor_enabled and self.winch.compressor_enabled)
```

Comment:
`teleopPeriodic` is called before components are. I'm not sure if this caused problems during comp, but yeah.